src/api/app/views/webui/users/tokens/index.html.haml

```diff
@@ -23,7 +23,10 @@
                 %td.text-word-break-all= link_to("#{truncate(token.package.project.name, length: 80)}/#{truncate(token.package.name, length: 80)}",
                   package_show_path(project: token.package.project, package: token.package)) if token.package
                 %td.text-center
-                  = link_to '#', title: 'Delete Token', class: 'btn btn-sm btn-outline-danger',
+                  - if policy([:webui, token]).edit?
+                    = link_to(edit_token_path(token), title: 'Edit Token') do
+                      %i.fas.fa-edit
+                  = link_to '#', title: 'Delete Token', class: 'btn btn-sm',
```

Comment:
:right_anger_bubble:  we could align the icons. But it's not necessarily required for this PR.

![Screenshot_2021-08-02 Open Build Service(2)](https://user-images.githubusercontent.com/2581944/127871153-942dea2f-fd0d-4481-8612-7f828bcf5c20.png)

Reply:
True, this is ugly! I'll look into this.

Reply:
Done, please have another look

Reply:
Better now.

Reply:
It could still be improved, like having a smaller column, but I would say this is out of scope for this PR.